src/herder/HerderImpl.cpp

```diff
@@ -1323,6 +1328,29 @@ HerderImpl::triggerNextLedger(uint32_t ledgerSeqToTrigger,
     if (nextCloseTime <= lcl.header.scpValue.closeTime)
```

Comment:
Is there any particular reason we only check if a clock is drifting behind and not drifting ahead?